README.md

```diff
@@ -1,102 +1,70 @@
-<h1 align="center" style="color:#2F3C7E">
-  Coinratecap.com
-</h1>
+# Getting Started with Create React App
```

Comment:
Shouldn’t we start with asking users to fork the project if they want to contribute? Also Avoid placing too many links,it causes distraction.

Reply:
Noted that